captain-definition

```diff
@@ -0,0 +1,17 @@
+{
```

Comment:
why is this included again? captain-definition file already exists in the root dir